docs/src/tutorials/creating_geometry.md

```diff
@@ -0,0 +1,303 @@
+I always think of a polygon as filled to the right of the lines as one progresses
+from point `n` to point `n+1`.
+
+````@example creating_geometry
+ring1 = GI.LinearRing(GI.getpoint(lines))
+````
+
+Now, let's make the `LinearRing` into a `Polygon`.
+
+````@example creating_geometry
+polygon1 = GI.Polygon([ring1]; crs)
+````
+
+Now, we can use GeometryOperations and CoordinateTransformations to shift `polygon1`
+up, to avoid plotting over our earlier results.  This is done through the [GeometryOps.transform](@ref) function.
+
+````@example creating_geometry
+xoffset = 0.
+yoffset = 50.
+f = CoordinateTransformations.Translation(xoffset, yoffset)
+polygon1 = GO.transform(f, polygon1)
+plot!(polygon1)
+fig
+````
+
+Polygons can contain "holes". The first `LinearRing` in a polygon is the exterior, and all 
+subsequent `LinearRing`s are treated as holes in the leading `LinearRing`.
+
+`GeoInterface` offers the `GI.getexterior(poly)` and `GI.gethole(poly)` methods to get the 
+exterior ring and an iterable of holes, respectively.
+
+!!! note
+    Some packages always consider the secondary `LinearRings` holes, others look at the winding
+    order, where the polygons are filled inward if they have a [clockwise](@ref GeometryOps.isclockwise) winding order and 
+    outward if they have a counterclockwise winding order.
+    
+    Hopefully, these are details that you'll never have to deal with.  But it is good to know.
+
+````@example creating_geometry
+hole = GI.LinearRing(GI.getpoint(multipoint))
+polygon1 = GI.Polygon([ring1, hole]; crs)
+````
+
+Shift `polygon1` to the right, to avoid plotting over our earlier results.
+
+````@example creating_geometry
+xoffset = 50.
+yoffset = 0.
+f = CoordinateTransformations.Translation(xoffset, yoffset)
+polygon1 = GO.transform(f, polygon1)
+plot!(polygon1)
+fig
+````
+
+`Polygon`s can also be grouped together as a `MultiPolygon`.
+
+````@example creating_geometry
+r = 5
+x = cos.(reverse(ϴ)) .* r .+ xoffset
+y = sin.(reverse(ϴ)) .* r .+ yoffset
+ring2 =  GI.LinearRing(GI.Point.(zip(x,y)))
+polygon2 = GI.Polygon([ring2])
+multipolygon = GI.MultiPolygon([polygon1, polygon2]; crs)
+````
+
+Shift `multipolygon` up, to avoid plotting over our earlier results.
+
+````@example creating_geometry
+xoffset = 0.
+yoffset = 50.
+f = CoordinateTransformations.Translation(xoffset, yoffset)
+multipolygon = GO.transform(f, multipolygon)
+plot!(multipolygon)
+fig
+````
+
+Great, now we can make `Points`, `MultiPoints`, `Lines`, `LineStrings`, `Polygons` (with holes), and `MultiPolygons` and modify them using [`CoordinateTransformations`] and [`GeometryOps`].
+
+But where does the `crs` information come in? To show this, we need to use `GeoMakie` that can interpret the `crs` information that we've included with our geometries.
+
+Now specify the source and destination projections for our map. Remember that the very first thing we did was select our source coordinate system.
+
+````@example creating_geometry
```

Comment:
I would explain what `source` and `dest` mean in this context right here. As someone who has never used this, I still don't really know what they mean (although you do have a note on the meaning of `source` way down the page).

Reply:
For GeoMakie, source is the CRS of the input and dest is the CRS you want to visualize in.  That could be made a bit clearer, I think - is there a particular verbiage you're thinking of?

Reply:
I think even that but of clarification is perfect.